src/components/ui/copy/copy.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import styled, { css } from 'styled-components';
+import { IconButton } from '../buttons/icon-button';
+import { Icon } from '../icon/icon';
+
+const Default_Clicked_Duration = 3000;
+
+interface CopyProps {
+  /**
+   * Duration, that the Checkmark will be shown once the button is clicked
+   * @default 3000ms
```

Comment:
This is super hard to maintain. Once you change the variable, the comment doesn't get updated automatically. It's probably better to just remove it. As the variable is also on top of the file it's already easy to spot the value.

Reply:
🧹